meteor_mongo_vx.x.x.js

```diff
@@ -17,6 +17,8 @@ declare class Meteor$Mongo$BulkOpHandle {
   updateOne(modifier: Meteor$Mongo$Modifier): void;
 }
 
+declare type OptionalId<T> = {|...$Exact<T>, _id: ?string|};
```

Comment:
What about `Meteor$Mongo$ObjectId`? I'd rather not separate this type. It's a useful one, but it pollutes types space.